jsonschema-validation.xml

```diff
@@ -740,6 +740,25 @@
                 </t>
             </section>
 
+            <section title='"readOnly"'>
+                <t>
+                    The value of this keyword MUST be a boolean.
+                </t>
+                <t>
+                    If it has a value of boolean true, this keyword indicates that the value of the
+                    instance is managed exclusively by the owning authority, and attempts by a user
+                    agent to modify the value of this property are expected to be ignored or
+                    rejected by that owning authority.
+                </t>
+                <t>
+                    For example, this property would be used to mark a server-generated serial
```

Comment:
Not the same as the other uses of server, but perhaps it would read better here if we said "database-generated" or "algorithmically generated" or something?